karaf/features/src/main/feature/feature.xml

```diff
@@ -332,9 +332,24 @@
         <bundle dependency="true">mvn:com.google.guava/guava/${guava.version}</bundle>
     </feature>
 
+    <!-- Don't load all bundles out of the box in the long run. Let users cherry pick what they need. -->
+    <feature name="brooklyn-software-all" version="${project.version}" description="Brooklyn All Software Entities">
+        <feature>brooklyn-software-base</feature>
+        <feature>brooklyn-jmxmp-agent</feature>
+        <feature>brooklyn-jmxrmi-agent</feature>
+        <feature>brooklyn-software-network</feature>
+        <feature>brooklyn-software-cm</feature>
+        <feature>brooklyn-software-osgi</feature>
+        <feature>brooklyn-software-database</feature>
+        <feature>brooklyn-software-webapp</feature>
```

Comment:
Longer term, should we move `karaf/*` to `brooklyn-dist` repo? I presume this will only work if we have pre-built `brooklyn-software-webapp` etc. It will presumably otherwise try to download it from apache nexus.